paddle/fluid/framework/attribute.h

```diff
@@ -165,6 +165,35 @@ struct ExtractAttribute<float> {
   const std::string& attr_name_;
 };
 
+template <>
+struct ExtractAttribute<std::vector<double>> {
+  explicit ExtractAttribute(const std::string& attr_name)
+      : attr_name_(attr_name) {}
+
+  std::vector<double>* operator()(Attribute& attr) const {
```

Comment:
Just knowledge, not your fault. Our C++ style requires the change-able input parameter `attr` as a pointer, but I read the old code doesn't follow the style...

So you don have to change anything. It is just a kindly remind comment :-)

Reply:
Thanks a lot.